pkg/sfu/buffer/rtpstats.go

```diff
@@ -782,6 +791,37 @@ func (r *RTPStats) GetRtcpSenderReport(ssrc uint32, srDataExt *RTCPSenderReportD
 		nowRTP = srDataExt.SenderReportData.RTPTimestamp + uint32(now.Sub(smoothedLocalTimeOfLatestSenderReportNTP).Milliseconds()*int64(r.params.ClockRate)/1000)
 	}
 
+	// TODO-REMOVE-AFTER-DEBUG
```

Comment:
marked this as remove after debugging, but we can potentially keep this and log largish drifts detected. In that case, need to make the the time stamp rollover proper.

Reply:
okay, implemented time stamp cycle counting in this [commit](https://github.com/livekit/livekit/pull/1669/commits/87bae6c7a85fa8e156307e2a32384383a25ca789)